MachUpskillingFY17/JabbR-Core
# invite code command working

To Test:

- Log on to JabbR in two separate browsers as two different users
- In one of the browsers, create a new room
- Go into the database and manually set the room as private in the ChatRooms table
- Restart JabbR. And you should see the new room has a little lock on it
- Go into the private room as both users on both browsers (the owner of the room should be able to get in, the other user should NOT be able to join)
- As the owner of the room , run the /invitecode  command. It should say "the invite code is [code]" (save this invite code somewhere or just copy it)
- As the other user in the other browser, in the lobby type /join [roomName] [code] and this user should now be able to join the private room. Once this new user is in the private room , see if they can run /invitecode. They should be able too and it should have the same results as earlier.
-Restart JabbR again , and test that both users can still join the private room and run the /invitecode command
